sig/generated/channels/application_cable/channel.rbs

```diff
@@ -1,3 +1,5 @@
+# Generated from app/channels/application_cable/channel.rb with RBS::Inline
```

Comment:
So far this comment headers from RBS::Inline were removed on subtracting. But it is kept generated from now on.